### PR TITLE
Session Replay is now in open beta

### PR DESCRIPTION
**Pull request by MirandaKapin**
https://docs-staging.datadoghq.com/MirandaKapin/SR-updates/real_user_monitoring/guide/session-replay-getting-started/

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/_index.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ Datadog's *Real User Monitoring (RUM)* gives you end-to-end visibility into the
 * **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage).
 * **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors).
 
+## What is Session Replay?
+
+Session Replay expands your UX monitoring by allowing you to capture and visually replay the web browsing experience of your users. Used in combination with RUM performance data, Session Replay can be an asset for error identification, reproduction, and resolution, and it can also bring invaluable insights into your web application’s usage patterns and design pitfalls. See here for a [guide to getting started with Session Replay][6].
```
Comment (alai97) [inline content/en/real_user_monitoring/_index.md:42]:
```suggestion
Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and identifies insights in your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][6].
```

File: content/en/real_user_monitoring/_index.md [hunk 2]
```diff
@@ -37,6 +37,13 @@ Datadog's *Real User Monitoring (RUM)* gives you end-to-end visibility into the
...
+
+<div class="alert alert-info"><p>Session Replay is currently in beta, there are no billing implications for the Session Replays during this period. At the moment, Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.
```
Comment (alai97) [inline content/en/real_user_monitoring/_index.md:44]:
```suggestion
<div class="alert alert-info"><p>Session Replay is in beta. There are no billing implications for your Session Replays during this period. Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.
```

File: content/en/real_user_monitoring/_index.md [hunk 3]
```diff
@@ -59,7 +66,7 @@ View user sessions in segments, such as checking when latency impacts your premi
 
 ### Integration with logs, APM, and profiler
 
-View your [backend traces, logs, infrastructure metrics][1] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.
+View your [backend traces, logs, infrastructure metrics][5] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.
```
Comment (alai97) [inline content/en/real_user_monitoring/_index.md:69]:
```suggestion
View your [backend traces, logs, and infrastructure metrics][5] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.
```

File: content/en/real_user_monitoring/guide/session-replay-getting-started.md [hunk 4]
```diff
@@ -12,7 +11,7 @@ further_reading:
   text: 'Use Datadog Session Replay to view real-time user journeys'
 ---
 
-<div class="alert alert-info"><p>Session Replay is in private beta. There are no billing implications for the Session Replays during this period. If you want to be added to the private beta, sign up by emailing <a href="mailto:support@datadoghq.com">support@datadoghq.com</a>.</p><p>Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a> at this time.</p>
+<div class="alert alert-info"><p>Session Replay is in beta. There are no billing implications for the Session Replays during this period. If you have any questions, please let us know by emailing <a href="mailto:support@datadoghq.com">support@datadoghq.com</a>.</p><p>Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a> at this time.</p>
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/session-replay-getting-started.md:14]:
```suggestion
<div class="alert alert-info"><p>Session Replay is in beta. There are no billing implications for the Session Replays during this period. If you have any questions, email <a href="mailto:support@datadoghq.com">support@datadoghq.com</a>.</p><p>Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.</p>
```

**Review by alai97 — APPROVED** [review]
Looks good pending the following commit!

File: content/en/real_user_monitoring/_index.md [hunk 5]
```diff
@@ -37,6 +37,13 @@ Datadog's *Real User Monitoring (RUM)* gives you end-to-end visibility into the
 * **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage).
 * **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors).
 
+## What is Session Replay?
+
+Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and identifies insights in your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][1].
```
Comment (cswatt) [inline content/en/real_user_monitoring/_index.md:42]:
```suggestion
Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][1].
```

File: content/en/real_user_monitoring/guide/session-replay-getting-started.md [hunk 6]
```diff
@@ -12,13 +11,14 @@ further_reading:
...
+<div class="alert alert-info"><p>Session Replay is in beta. There are no billing implications for the Session Replays during this period. If you have any questions, email <a href="mailto:support@datadoghq.com">support@datadoghq.com</a>.</p><p>Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.</p>
 </div>
 
 ## What is Session Replay?
 
-Session Replay expands your UX monitoring by allowing you to capture and visually replay the web browsing experience of your users.
-Used in combination with RUM performance data, Session Replay can be an asset for error identification, reproduction, and resolution, and it can also bring invaluable insights into your web application’s usage patterns and design pitfalls.
+Session Replay expands your user experience monitoring by allowing you to capture and visually replay the web browsing experience of your users.
+
+Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and identifies insights in your web application’s usage patterns and design pitfalls.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/session-replay-getting-started.md:21]:
```suggestion
Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**